app/Http/Controllers/ApplicationByJobController.php

```diff
@@ -18,8 +18,7 @@
 use App\Models\SkillDeclaration;
 use App\Models\WorkExperience;
 use App\Services\Validation\ApplicationValidator;
-use Facades\App\Services\WhichPortal;
-use Illuminate\Http\Request;
```

Comment:
Submitting an application broke. Looks like `use Illuminate\Http\Request` is used in `submit()` and `updateAssetSkills()` method.